### PR TITLE
added muleify

### PR DESCRIPTION
**Pull request by AlexanderElias**

### CODE REVIEW

File: source/projects/muleify.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Muleify
+repo: AlexanderElias/muleify
+homepage: https://github.com/AlexanderElias/muleify
+language: JavaScript
+license: MPL-2.0
+templates: Comments
+description: "A zero configuration static site generator and asset preprocessor compiler."
+---
+
+# Muleify #
```
Comment (bdougie) [inline source/projects/muleify.md]: Can you remove this title. It gets duplicated, see: https://deploy-preview-226--staticgen.netlify.com/muleify

screenshot 2017-03-01 21 53 56

**Comment by AlexanderElias** [comment]
Oh weird yes I will get right on that.

**Comment by bdougie** [comment]
Looks like the CI failed, can you also rebase the latest from master onto your branch. I believe that will fix it.

**Comment by AlexanderElias** [comment]
There we go, I believe we are all fixed.

**Comment by bdougie** [comment]
Thanks

**Pull request merged**